src/core/selinux-access.c

```diff
@@ -30,13 +31,13 @@ static bool initialized = false;
 struct audit_info {
         sd_bus_creds *creds;
         const char *path;
-        const char *cmdline;
         const char *function;
 };
 
 /*
    Any time an access gets denied this callback will be called
    with the audit data.  We then need to just copy the audit data into the msgbuf.
+   Total audit buffer length is 1024.
```

Comment:
what does this mean? which buffer does this refer to? are you suggesting msgbufsize is 1k typically? please be explicit if so